de/manual/ch_Tools_Assistants.xml

```diff
@@ -73,33 +73,48 @@
+        <variablelist>
+          <varlistentry>
+            <term>Ein einzelnes <emphasis>Konto</emphasis> durchsuchen</term>
 
-      <para>Bei den Buchungsfeldern werden nur die Buchungen berücksichtigt, die mit einem Buchungsteil in
-        diesem Register verknüpft sind, während bei den übrigen Buchungen nur die mit dem Konto
-        verbundenen Buchungsteile durchsucht werden, die diesem Konto zugeordnet sind, außer Sie
-        verwenden die Suchfunktion <guilabel>Alle Konten</guilabel>.
+            <listitem>
+              <para>Führen Sie die Suche in der Registerkarte eines Kontos durch, so werden nur die zu diesem Konto
+                zugehörigen Buchungsteile durchsucht. Möchten Sie doch eine Suche über mehrer
+                Konten durchführen, verwenden Sie als Suchkriterium <guilabel>Alle
+                Konten</guilabel>.
+                <footnote id="fn_tools_all_accounts">
+                  <para>Das Suchkriterium <guilabel>Alle Konten</guilabel> führt eine Suche durch, bei der Buchungen in den
+                    Konten, die im dem Dialog, nach betätigen der Schaltfläche <guilabel>Konten
+                    wählen</guilabel>, markiert wurden, gefunden werden und mindestens einen
+                    Buchungsteil in jedem der ausgewählten Konten enthalten. Wenn die Suche in
+                    einem regulären Kontobuch gestartet wird, ist dies das einzige Kriterium, das
+                    Buchungsteile von anderen Konten prüft. Wenn Sie also den Dialog
+                    <guilabel>Buchungen suchen</guilabel> z.B. in Aktiva:Umlaufvermögen:Girokonto
+                    öffnen und alle Buchungen dieses Kontos mit einem Buchungsteil in
+                    Ausgaben:Lebensmittel finden wollen, ist dies als Kriterium zu verwenden.
+                    Beachten Sie jedoch, dass es sich um eine <emphasis>UND</emphasis>-Suche
+                    handelt: Wenn Sie zusätzlich auch nach Ausgaben:Verschiedenes suchen, erhalten
+                    Sie <emphasis>nur</emphasis> die Buchungen mit einem Buchungsteil in jedem der
+                    beiden Konten und <emphasis>nicht</emphasis> die Buchungen mit nur einem
+                    einzigen Buchungsteil.
+                  </para>
+                </footnote>
+              </para>
+            </listitem>
+          </varlistentry>
+
+          <varlistentry>
+            <term>Das gesamte <emphasis>&appname;-Buch</emphasis> durchsuchen</term>
+
+            <listitem>
+              <para>Starten Sie die Suche entweder in der <emphasis>Kontenübersicht</emphasis>, oder in dem geöffneten
+                <xref linkend="general-journal" />, so wird die Suche über alle Konten ausgeführt.
+              </para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
       </para>
 
       <para>Wenn Sie zuvor die Buchungen in einem Register mit &mc.vw.f-by; filtern, dann werden in diesem Konto
```

Comment:
Am Ende des  Absatzes:
Statt Klammern würde ich für Einschübe generell Spiegelstriche — verwenden.
Ich finde, das ist agenehmer zu lesen und sieht professioneller aus.
vielleicht mal mit grep … und sed …; drüber gehen?